contrib/inventory/ec2.py

```diff
@@ -491,9 +491,14 @@ def get_rds_instances_by_region(self, region):
         try:
             conn = self.connect_to_aws(rds, region)
             if conn:
-                instances = conn.get_all_dbinstances()
-                for instance in instances:
-                    self.add_rds_instance(instance, region)
+                marker = None
+                while True:
+                    instances = conn.get_all_dbinstances(marker=marker)
+                    marker = instances.next_marker
```

Comment:
@bcoca I see no reference to `next_marker` in the [official documentation](http://boto.cloudhackers.com/en/latest/ref/rds.html), only `marker`.

Your code's logic is correct: you do a first request, then you get a marker, then you do another request with the marker you obtained previously. But `next_marker` is always empty, while `marker` gives you the correct information. I've tested it and it works.

I think that boto/boto#3109 is invalid because we do get a marker in the `marker` attribute of the returned object.